x/metadata/keeper/msg_server_test.go

```diff
@@ -67,11 +63,1015 @@ func TestMsgServerTestSuite(t *testing.T) {
 //   - If errorString is empty, theError must be nil
 //   - If errorString is not empty, theError must equal the errorString.
 func (s *MsgServerTestSuite) AssertErrorValue(theError error, errorString string, msgAndArgs ...interface{}) bool {
-	return AssertErrorValue(s.T(), theError, errorString, msgAndArgs...)
+	return assertions.AssertErrorValue(s.T(), theError, errorString, msgAndArgs...)
+}
```

Comment:
Line 25 of the file:
```
	"github.com/provenance-io/provenance/testutil/assertions"
```

That's in the import section.

You can also tell that it's being imported because it compiles and all the tests pass.